topologies/kne/arista_ceos_otg.textproto

```diff
@@ -1,6 +1,6 @@
 name: "ixia-c"
 nodes: {
-    name: "ate"
+    name: "otg"
```

Comment:
I think OTG tests still use `ondatra.ATE(t, "ate")` not `ondatra.ATE(t, "otg")`. Would this cause problem?

https://github.com/openconfig/featureprofiles/blob/317a6776ef8844d31bcece10c4d50621e6814bb0/feature/interface/singleton/otg_tests/singleton_test/singleton_test.go#L517

Reply:
No, because the Ondatra names are from the testbed, not the KNE topology